feat/summary-tags-events (#3864)
getsentry/sentry-docs: src/docs/product/performance/transaction-summary.mdx

Frontend transactions will have a "Web Vitals" tab. Clicking on the tab will take you to the [Web Vitals page](/product/performance/web-vitals), where you can see a detailed view of the web vitals associated with this transaction.

## Tags

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

> On this tab, suspect tag keys that often correspond to slower transactions are listed. Toggle through them to have corresponding tag values reflected in the heat map. Similar to a histogram, events are distributed by duration. Click on any purple box to view a list of events or continue to filter down the events by targeting values in the table below the heat map.

Reviewer: The last sentence seemed a bit vague to me, so I've made a suggestion. Please make sure my edit hasn't changed the meaning.

```suggestion
The **Tags** tab displays a list of suspect tag keys that often correspond to slower transactions. Toggle through them to have the corresponding tag values reflected in the heat map. Similar to a histogram, events are distributed by duration. Click on any purple box to view a list of events or continue to filter down the events by  adding values to the filter in the table below the heat map.
```

> The **Tags** tab displays a list of suspect tag keys that often correspond to slower transactions. Toggle through them to have the corresponding tag values reflected in the heat map. Similar to a histogram, events are distributed by duration. Click on any purple box to view a list of events or continue to filter down the events by  adding values to the filter in the table below the heat map.

Reviewer: I introduced an extra space with my edit. Just fixing that.

```suggestion
The **Tags** tab displays a list of suspect tag keys that often correspond to slower transactions. Toggle through them to have the corresponding tag values reflected in the heat map. Similar to a histogram, events are distributed by duration. Click on any purple box to view a list of events or continue to filter down the events by adding values to the filter in the table below the heat map.
```

![Example of Tags Tab](./tags-tab.png)

> ## Events

Reviewer: Assuming the name of this tab isn't going to change, all of the other headings for tabs are the same as the name of the tab.

```suggestion
## All Events
```

<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

> On this tab, the table offers a full list of transactions broken down similar to the abbreviated table in the overview. Filters are carried across from other views. You can also narrow the events down by percentile.

Reviewer: Do you mean "broken down in a similar way as the abbreviated table" or is "broken down" a descriptor by itself? If the second, what does that mean exactly?

```suggestion
The table in **All Events** tab offers a full list of transactions broken down similar to the abbreviated table in the overview. Filters are carried across from other views. You can also narrow the events down by percentile.
```

## Additional Actions

> ### Starring Transactions

Reviewer: Are we moving away from the "key transactions" language? What about "Creating Key Transactions" here?
